### PR TITLE
NGSTACK-714: building the embedded view should not crash the parent view

### PR DESCRIPTION
**Pull request by pspanja**

### CODE REVIEW

File: bundle/View/ContentRenderer.php [hunk 1]
```diff
@@ -77,7 +74,11 @@ public function renderEmbeddedContent(string $viewType, array $parameters = []):
             '_controller' => 'ng_content:embedAction',
         ];
 
-        $view = $this->viewBuilder->buildView($baseParameters + $parameters);
+        try {
+            $view = $this->viewBuilder->buildView($baseParameters + $parameters);
+        } catch (Exception $exception) {
```
Comment (emodric) [inline bundle/View/ContentRenderer.php]: Why not catch `Throwable`, if you're catching a generic exception?

File: bundle/View/ContentRenderer.php [hunk 2]
```diff
@@ -77,7 +74,11 @@ public function renderEmbeddedContent(string $viewType, array $parameters = []):
...
+            return '';
```
Comment (emodric) [inline bundle/View/ContentRenderer.php:89]: Could be good idea to write something to log in debug mode?

Comment (pspanja) [reply bundle/View/ContentRenderer.php:89]: Yes, added in https://github.com/netgen/ibexa-site-api/pull/12/commits/0f9a10cbf814aeb23762d65d072c9e5b2a32d023

File: bundle/View/ContentRenderer.php (hunk 1 above)
Comment (pspanja) [reply bundle/View/ContentRenderer.php]: Added in https://github.com/netgen/ibexa-site-api/pull/12/commits/0f9a10cbf814aeb23762d65d072c9e5b2a32d023

**Comment by pspanja** [comment]
Thanks @emodric

**Pull request merged**